lib/fathead/npm_cli/parse.py

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import requests
+import re
+from bs4 import BeautifulSoup
+
+
+START_URL = "https://docs.npmjs.com/cli/access"
```

Comment:
Keeping the fetch and parse scripts separate helps if you need to update either (eg. website structure changes) without touching the other. Other advantages is that you can regenerate the output.txt without re-fetching the sources (even if there's no Internet connection); 
The code is well done and looks good, though. 
What you think @moollaza?